crates/sui-framework/packages/bridge/sources/message.move

```diff
@@ -82,11 +83,33 @@ module bridge::message {
         token_prices: vector<u64>
     }
 
+    // For read
+    public struct ParsedTokenTransferMessage has drop {
+        message_version: u8,
+        seq_num: u64,
+        source_chain: u8,
+        payload: vector<u8>,
+        parsed_payload: TokenTransferPayload,
+    }
+
+    // Convert BridgeMessage to ParsedTokenTransferMessage
+    public fun to_parsed_token_transfer_message(message: &BridgeMessage): ParsedTokenTransferMessage {
```

Comment:
`public`? it may very weel be fine and harmless no matter what but figured I'd ask